src/DIRAC/WorkloadManagementSystem/Client/JobStatus.py

```diff
@@ -105,7 +105,7 @@ def __init__(self, state):
                                       [WAITING, RECEIVED, DELETED],
                                       defState=RESCHEDULED),
                    MATCHED: State(5,
-                                  [RUNNING, KILLED],
+                                  [RUNNING, FAILED, KILLED],
```

Comment:
that seem to be quite an important change no ? Does it deserve to be in the wiki ?